src/styles/blocks/calendar.scss

```
@@ -0,0 +1,72 @@
+$dayWidth: 100px;
+$dayGap: 1px;
+
+body {
+  margin: 0;
+}
+
+.page__calendar {
+  margin: 0 auto;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 30px;
+  gap: 1px;
+  width: 7 * $dayWidth + 6 * $dayGap;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    width: $dayWidth;
+    height: $dayWidth;
+
+    box-sizing: border-box;
+    border: 1px solid black;
+    background-color: #eee;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+
+    cursor: pointer;
+    transition: transform 0.5s;
+  }
+
+  &__day:hover {
+    background-color: pink;
+    transform: translateY(-20px);
+  }
+
+  @for $day from 1 through 31 {
+    &__day--#{$day}::after {
+      content: "#{$day}";
+
+    }
+  }
+
+  $week: ("mon": (0),
+  "tue": ($dayWidth + $dayGap),
+  "wed": (2 * $dayWidth + 2 * $dayGap),
```

you can calculate it just one time instead calculate each time
```suggestion
  }
  $day-margin: $dayWidth + $dayGap;
  
  $week: ("mon": (0),
  "tue": $day-margin,
  "wed": 2 * $day-margin,
```

---

move vars to the separate file